workflow/commit-messages.md

```diff
@@ -0,0 +1,43 @@
+---
+title: Commit guidelines
+description: Some general guidelines to help you in writing good commits for collaboration and clarity.
+---
+
+# Commit Guidelines
```

Comment:
Actually, if you have `title: Commit guidelines`, I think you should remove this heading, 'cos I think it will look duplicated.

You can see how it looks with `gem install jekyll jekyll-redirect-from; jekyll serve`, and also see [other documents](https://github.com/canonical-webteam/practices/blob/master/project-structure/server-side-frameworks.md) for examples.

Reply:
Good catch. Wrote this outside the repo first.